compile failture on Mac

cherrytree/future/src/ct/ct_actions_import.cc:234:24: error: 
      constructor inherited by 'CtZimImportHandler' from base class 'CtParser'
      is implicitly deleted
    CtZimImportHandler handler(_pCtMainWin->get_ct_config());
                       ^
cherrytree/future/src/ct/ct_imports.h:424:51: note: 
      constructor inherited by 'CtZimImportHandler' is implicitly deleted
      because base class 'CtTextParser' has a deleted corresponding constructor
class CtZimImportHandler: public CtImportHandler, public CtTextParser

--- a/future/src/ct/ct_import_handler.cc
+++ b/future/src/ct/ct_import_handler.cc
@@ -21,7 +21,7 @@
 #include "ct_imports.h"
 #include "ct_const.h"
 #include "ct_config.h"
-#include <fmt/format.h>
+#include <fmt/fmt.h>
 
 namespace fs = std::filesystem;
 

--- a/future/src/ct/ct_imports.h
+++ b/future/src/ct/ct_imports.h
@@ -440,7 +440,7 @@ protected:
     
     std::vector<std::shared_ptr<CtImportFile>> _get_files(const std::filesystem::path& path, uint32_t current_depth, CtImportFile* parent);
 public:
-    using CtImportHandler::CtImportHandler;
+    CtZimImportHandler(CtConfig* config) : CtParser(config), CtTextParser(config) {}
 
     void feed(std::istream& data) override;
     
@@ -460,7 +460,7 @@ protected:
     
     bool _in_link = false;
 public:
-    using CtTextParser::CtTextParser;
+    CtMDParser(CtConfig* config) : CtParser(config), CtTextParser(config) {}
 
     void feed(std::istream& stream) override;
 

--- a/future/src/ct/ct_parser.cc
+++ b/future/src/ct/ct_parser.cc
@@ -22,7 +22,7 @@
 #include "ct_imports.h"
 #include "ct_const.h"
 #include "ct_config.h"
-#include <fmt/format.h>
+#include <fmt/fmt.h>
 #include <iostream>
 #include <memory>
 

--- a/future/src/ct/ct_text_parser.cc
+++ b/future/src/ct/ct_text_parser.cc
@@ -21,7 +21,8 @@
 
 #include "ct_imports.h"
 #include <iostream>
-#include <src/fmt/format.h>
+
+#include <fmt/fmt.h>
 
 template<class T>
 std::unordered_set<char> build_pos_tokens(const T& tokens) {

--- a/future/src/ct/ct_types.h
+++ b/future/src/ct/ct_types.h
@@ -125,7 +125,7 @@ struct CtStorageSyncPending
     std::set<gint64>                               nodes_to_rm_set;
 };
 
-class CtNodeData;
+struct CtNodeData;
 class CtAnchoredWidget;
 class CtStorageEntity
 {

